apis/apiextensions/v1beta1/zz_generated.composition_revision_types.go

```diff
@@ -134,8 +134,11 @@ type CompositionRevisionStatus struct {
 // +genclient
 // +genclient:nonNamespaced
 
-// A CompositionRevision represents a revision in time of a Composition.
-// Revisions are created by Crossplane; they should be treated as immutable.
+// A CompositionRevision represents a revision of a Composition. Crossplane
+// creates new revisions when there are changes to the Composition.
+//
+// Crossplane creates and manages CompositionRevisions. Composition Revisions
+// aren't designed for user changes.
```

Comment:
Instead of "aren't designed for user changes" could we be more forceful? Something like "You shouldn't edit Composition Revisions directly". (Ditto for all the other revision types.)